environment/postInstall

```diff
@@ -1,5 +1,16 @@
 #!/usr/bin/env bash
 set -e
 
+# install spikeinterface extras
+pip install --no-cache-dir --ignore-installed spikeinterface[full,widgets]==0.98.0
+
+# install aind-data-schema from source
+git clone https://github.com/AllenNeuralDynamics/aind-data-schema.git
+cd aind-data-schema
+git checkout f5d3f3ee31bf59077b16fd17211104708055e3f6
```

Comment:
It'd be a little better if we waited until the latest update in aind-data-schema gets merged into main...

Reply:
Of course, this was just for testing ;) We'll wait for the release of `aind-data-schema`